brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2055 Slider Arrow Padding Option

### Description
Added Arrow Padding option for slider block. 

### Screenshots
![Edit Page “Slider demo” ‹ WordPress — WordPress 2022-11-16 at 5 06 09 PM](https://user-images.githubusercontent.com/10000232/202170530-bbe8c491-a96c-4d42-98e2-f54511e3036e.jpg)

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?

1. Add a Slider block in the page/post. 
2. Go to Style -> Arrow and Dots -> Padding 
3. You can set a responsive option also. 
4. Check on the front end and backend preview also. 
5.   For better design, padding has been applied as width and height and calculated as `( Padding * 2  ) + Arrow Size` 

### Checklist:
